### PR TITLE
Added more explicit instructions for BrowserSync.

### PR DESCRIPTION
**Pull request by JulienMelissas**
Hopefully we get less of these threads: https://discourse.roots.io/t/browser-sync-wordpress-and-mamp/3061/7

### CODE REVIEW

**Comment by JulienMelissas** [comment]
@kalenjohnson
http://38.media.tumblr.com/tumblr_m9t31sX4ER1qgcra2o1_500.gif

File: README.md [hunk 1]
```diff
@@ -83,8 +83,27 @@ You now have all the necessary dependencies to run the build process.
...
+  "config": {
+    "devUrl": "project-name.dev"
+  }
+...
+```
+If your local develoment url looks like `http://localhost:8888/project-name/` you would update the file to read:
+```json
+...
+  "config": {
+    "devUrl": "http://localhost:8888/project-name/"
```
Comment (JulienMelissas) [inline README.md]: Confirmed this is working with a MAMP project

File: README.md [hunk 2]
```diff
@@ -83,8 +83,27 @@ You now have all the necessary dependencies to run the build process.
 * `gulp watch` — Compile assets when file changes are made
 * `gulp --production` — Compile assets for production (no source maps).
 
+### Using BrowserSync
+
 To use BrowserSync during `gulp watch` you need update `devUrl` at the bottom of `assets/manifest.json` to reflect your local development hostname.
 
+For example, if your local development url is `http://project-name.dev` you would update the file to read:
```
Comment (retlehs) [inline README.md]: url -> URL

File: README.md [hunk 3]
```diff
@@ -83,8 +83,27 @@ You now have all the necessary dependencies to run the build process.
...
+```json
+...
+  "config": {
+    "devUrl": "http://project-name.dev"
+  }
+...
+```
+If your local develoment url looks like `http://localhost:8888/project-name/` you would update the file to read:
```
Comment (retlehs) [inline README.md]: url -> URL

**Pull request merged**